demo/app/main-view-model.ts

```diff
@@ -1,10 +1,11 @@
-import { Observable } from 'tns-core-modules/data/observable';
-import { EventData } from 'tns-core-modules/ui/core/view';
-import { CastEvent, CastMediaInfo, CastMediaStatus } from 'nativescript-cast/cast.types';
+import {Observable} from 'tns-core-modules/data/observable';
```

Comment:
Can we keep the spaces around the brackets `{ Observable }`. It's probably your linter formatting it automatically.